lib/zfs.js

```diff
@@ -307,6 +307,30 @@ function clone(opts, cb) {
 
 /*
 *
+*  Mount the specified dataset to the mountpoint (have been set before mount)
+*
+*  PARAMS:
+*  opts: {
+*    dataset: string // the name of the zfs dataset (must exist)
+*    overlay: boolean // whether use overlay mode
+*  }
+ */
+function mount(opts, cb) {
+    "use strict";
+    
+    var params = [ 'mount' ];
+    
+    if (opts.overlay) {
+        params.push('-O');
+    }
+    
```

Comment:
Yes, of course.

Actually, I just don't know the detail of the -o options:)

In my project, I just set the mountpoint before mount cmd.

I'll improve my code and pull another request